format/src/main/thrift/schema.thrift

```diff
@@ -135,9 +135,9 @@ struct ColumnSchema{
 	17: optional list<ParentColumnTableRelation> parentColumnTableRelations;
 
   /**
-   * To specify if it is an index column. Its Default value is false
+   * To specify if it is an spatial column. Its Default value is false
 	 */
-	18: optional bool indexColumn;
+	18: optional bool spatialColumn;
```

Comment:
Is this the `sourcecolumns` of the spatial index?
If yes
1. mention it in the description.
2. change name to `isSpatialSourceColumn`

Reply:
This property is set to true for the spatial index column. Not for the source columns. Spatial index column is a non schema column created internally with SPATIAL_INDEX property. Source columns are normal table schema columns.